README.md

```diff
@@ -482,6 +483,33 @@ class { '::sensu::backend':
 }
 ```
 
+### Composite Names for Namespaces
+
+All resources that support having a `namespace` also support a composite name to define the namespace.
+
+For example, the `sensu_check` with name `check-cpu in qa` would be named `check-cpu` and put into the `qa` namespace.
+
+Using composite names is necessary if you wish to have multiple resources with the same name but in different namespaces.
+
+For example to define the same check in two namespaces using the same check name:
+
+```puppet
+sensu_check { 'check-cpu in default':
```

Comment:
if you don't specify a composite name would the namespace be specified in the type/provider as `default` or would it be unspecified?

Reply:
The default `namespace` is always `default` so if the name is just `check-cpu` the namespace is `default` if not specified.  I put the default in the name as thought it'd be confusing to mix non-composite and composite in the same example.